Makefile

```diff
@@ -73,7 +73,7 @@ else
 			HOST_ARCH = x86
 		endif
 		ifeq ($(shell uname -m),armv7l)
-			HOST_ARCH = armv7l
+			HOST_ARCH = armhf
```

Comment:
IMHO we should keep this 'top-level' ARCH string as `armv7hf`, as that's much more specific, and then convert it as necessary inside the other scripts.